_posts/articles/2022-08-16-genart.md

```diff
@@ -0,0 +1,28 @@
+
+# AI Village Art Day @ DEF CON 30
+Today, models that appear to generate novel outputs conditioned on text are becoming the mainstay of popular culture.
+New groups of users are starting to adopt generative architectures and bring new modes (experiential, creative) of interacting with large models.
```

Comment:
What are you trying to say here- can you give more detail (examples of groups, what architectures exist, what are these modes of interaction). Probably this should be expanded to a full paragraph